test/test_model.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+import datetime
+
+def check_file_exists(filename):
+    return os.path.isfile(filename)
+
+def test_model(model_name, model_dir):
+    """
+    Tests the model for candle compliance
+    """
+
+
+    print("Test start", datetime.datetime.now())
+    print("model_name:", model_name)
+    print("model_dir:", model_dir)
+    
+
+    # Check python version
+    python_version = sys.version_info.minor 
+    if python_version < 3:
+        python = "python3"
+    else:
+        python = "python"
+
+    if model_name:
+        # NOTE: file names are expected to be lowercase eg. graphdrp_baseline_pytorch.py
+        model_name_lower = model_name.lower()
+        # List files in model_dir
+        files_in_model_dir = os.listdir(model_dir)
+        baseline_pattern = model_name_lower + "_baseline"
+        config_pattern = model_name_lower + "_default_model"
+        
+        # for debugging
+        print(files_in_model_dir, "files_in_model_dir")
+        
+        baseline = None
+        config = None
+        
+        try:
+            baseline = [f for f in files_in_model_dir if baseline_pattern in f][0]
+            config = [f for f in files_in_model_dir if config_pattern in f][0]
+        except IndexError:
+            print("No baseline or config file found")
+            
+        print("baseline / config:", baseline, config)
+        if baseline:
+            if check_file_exists(baseline):
```

Comment:
This check is not included with the config check on line 55. Note that the check_file_exists function will fail since the `baseline` does not include the full path to the file.

Reply:
The path is take care of in earlier lines. The problem is the case of model name. I will push a fix that will compare both names in lower case.